frontend/messages/it.json

```
@@ -248,247 +253,253 @@
 	"remediationPlan": "Piano di rimedio",
-	"treatmentPlan": "Piano del trattamento",
+	"treatmentPlan": "Piano di trattamento",
 	"plan": "Piano",
-	"asPDF": "in PDF",
-	"asCSV": "in CSV",
+	"asPDF": "come PDF",
+	"asCSV": "come CSV",
+	"asWord": "come Word",
 	"draft": "Bozza",
-	"riskMatrixView": "Vista matrice di rischio",
+	"riskMatrixView": "Vista della matrice di rischio",
```

Biome (1.9.4)

[error] The key draft was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)